src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -213,7 +213,8 @@ internal BinaryClassificationTrainers(BinaryClassificationCatalog catalog)
         /// <summary>
         /// Run cross-validation over <paramref name="numberOfFolds"/> folds of <paramref name="data"/>, by fitting <paramref name="estimator"/>,
         /// and respecting <paramref name="samplingKeyColumnName"/> if provided.
-        /// Then evaluate each sub-model against <paramref name="labelColumnName"/> and return metrics.
+        /// Then evaluate each sub-model against <paramref name="labelColumnName"/> and return <see cref="BinaryClassificationMetrics"/>, which
```

Comment:
>evaluate each sub-model [](start = 17, length = 23)

Can you also mention that each sub-model is evaluated on the fold of data that it did not see during training?